src/Dapr.Client/TypeConverters.cs

```diff
@@ -12,7 +12,7 @@ namespace Dapr.Client
     /// <summary>
     /// Some type converters.
     /// </summary>
-    internal static class TypeConverters
+    public static class TypeConverters
```

Comment:
We should not make this public. These are helpers for use inside the client code. We should not be providing opinionated general purpose APIs like this as part of Dapr.

Reply:
fixed